docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml

```diff
@@ -293,7 +293,13 @@ example: |-
       "broker": {
         "type": ["Organization"],
         "name": "Cole United",
-        "leiCode": "54321351219389121979"
+        "externalIdentifiers": [
+          {
+            "type": ["ExternalIdentifier"],
+            "name": "LEI Code",
+            "value": "OHVJ56TBUFWT9XWMBEJ9"
```

Comment:
In JSON-LD / RDF... you basically always have name value pairs...

the name is the "property", like 

`identifier` which gets expanded to an IRI like: https://schema.org/identifier

The value, is a value for the property and can have various legal RDF types associated.

https://www.w3.org/TR/json-ld11/#typed-values

Here you are making a new RDF Class "ExternalIdentifier", for a key value pair...

Its sorta not the data modeling convention JSON-LD / RDF experts might expect...

Here is a similar example from schema.org

```json
{
 "@context": "https://schema.org/",
 "@type": "Book",
 "name": "Library linked data in the cloud : OCLC's experiments with new models of resource description",
 "author": "Carol Jean Godby",
 "isbn": "9781627052191",
 "identifier": {
 "@type": "PropertyValue",
   "propertyID": "OCoLC",
   "value":  "889647468"
  },
 "sameAs": "http://www.worldcat.org/oclc/889647468"
}
```

Should we use schema.org instead?

Reply:
Sorry, not entirely following.

I'd be happy to find an alternative to creating an additional schema for such a simple use case, but I'm not clear on how else I could define the `Organization.yml` `externalIdentifiers` prop.

I see that `schema.org/identifier` accepts a `PropertyValue` type which seems like it'd work for our use case, but is there any way to define a property to use an external schema?

For reference, the current `externalIdentifiers` prop on `Organization` that'd need to be modified:
```
  externalIdentifiers:
    title: External Identifiers
    description: External identifiers for this organization.
    items:
      $ref: ./ExternalIdentifier.yml
    $linkedData:
      term: externalIdentifiers
      '@id': https://schema.org/identifier

```

Reply:
`externalIdentifiers` is an RDF Predicate.

`ExternalIdentifier` is an RDF Type.


I question why we are defining either of these instead of using an existing term from schema.org or elsewhere.